server.js

```diff
@@ -3,12 +3,12 @@ const exphbs = require('express-handlebars');
 
 const app = express();
 const PORT = process.env.PORT || 8080;
-const home = require('./router/home'); // rename this to router
-const about = require('./router/about'); // rename this to router
-const contact = require('./router/contact'); // rename this to router
+const home = require('./routes/home'); // rename this to router
+const about = require('./routes/about'); // rename this to router
+const contact = require('./routes/contact'); // rename this to router
 
 app.use(express.static('public'));
-app.engine('.hbs', exphbs({extname: '.hbs', defaultLayout: 'main'}));
+app.engine('.hbs', exphbs({ extname: '.hbs', defaultLayout: 'main' }));
 app.set('view engine', '.hbs');
 
 app.get('/', (req, res) => {
```

Comment:
This should have been moved below the three `app.use` invocations below. This actually prevents the home page from actually rendering anything more than smoke test.